proposals/1/loadSet.md

```diff
@@ -0,0 +1,129 @@
+# Load Set
+
+## Current situation
+
+-  The hardcoded path for mod installation or discovery is `assets/mods/`. The mod must exist there to be found. 
+- Big mods that may conflict have to be individually disabled to minimize issues.
+- It is harder to find source of problem with big mods potentially messing with smaller mods.
```

Comment:
`Current situation` should not reference `Solutions` i.e. replace `harder` with `hard`